diff --git a/src/components/FormEditor.vue b/src/components/FormEditor.vue
--- a/src/components/FormEditor.vue
+++ b/src/components/FormEditor.vue
@@ -241,7 +241,7 @@ onBeforeMount(() => {
         if (
             typeof config.value.show_all_fields == 'boolean' &&
             config.value.show_all_fields
         ) {
-            show_all_fields = true;
+            show_all_fields.value = true;
         }
     }
